dist/api_data.py

```diff
@@ -1735,6 +1731,10 @@ def __ge__(self, other):
         choices=['read-uncommitted', 'read-committed', 'snapshot']),
     Config('name', '', r'''
         name of the transaction for tracing and debugging'''),
+    Config('no_timestamp', 'false', r'''
```

Comment:
I think this is true of all txn flags, but just double checking: this flag is cleared when a transaction is committed/rolled back? And when the parent session is reset?

Reply:
Yes; it's reinitialized for each new transaction.